.gitignore

```diff
@@ -1,24 +1,34 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
-*.pyc
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
 
-hg-data/histogram_tools.py
+# testing
+/coverage
 
-hg-data/Histograms.json
+# production
+/build
 
-hg-data/nsDeprecatedOperationList.h
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
 
-hg-data/UseCounters.conf
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
 
+# pre-react ignores
+*.pyc
+hg-data/histogram_tools.py
+hg-data/Histograms.json
+hg-data/nsDeprecatedOperationList.h
+hg-data/UseCounters.conf
 hg-data/dom/base/usecounters.py
```

Comment:
Maybe all of *hg-data*?

Reply:
I initially wanted to preserve these since I imagine georg's local repo has them but I wouldn't be surprised if they are an old artifact. Will remove and re-add if requested.